src/nova-driver.md

```diff
@@ -0,0 +1,71 @@
+# Nova GPU Driver
+
+Nova is a driver for GSP-based Nvidia GPUs that is currently under development
+and is being written in Rust.
+
+Currently, the objective is to upstream Rust bindings / abstractions for the
+relevant subsystems a prerequisite for the actual driver. Hence, the first
+mainline version of Nova will be a stub driver which helps establishing the
+necessary infrastructure in other subsystems (notably PCI and DRM).
+
+## Contact
+
+To contact the team and / or participate in development, please use the mailing
+list: nouveau@lists.freedesktop.org
+
+
+## Resources
+
+- [Official Source Tree](https://gitlab.freedesktop.org/drm/nova)
+- [Announcement E-Mail](https://lore.kernel.org/dri-devel/Zfsj0_tb-0-tNrJy@cassiopeiae/)
+
+In the source tree, the driver lives in `drivers/gpu/drm/nova`
+
+
+## Roadmap
+
+Nova's development can be distinguished into two different main steps:
+
+- First, a stub driver will be upstreamed into DRM. This driver solely serves
+  as a preparation step for the actual driver (see below). "Nova-stub" will
+  implement basic calls into subsystem code (mainly PCI and DRM) to establish a
+  user for their code base. It, therefore, serves as a user to lift these
+  bindings into the kernel.
+- Once all necessary subsystem code has been upstreamed successfully,
+  development of the actual driver will begin. "Nova-stub" will then
+  successively be replaced with "real Nova".
```

Comment:
After the initial life we'll surely need a whole lot more bindings that need to be written / upstreamed. The main idea is to continuously develop the driver upstream with the stub driver doing the initial lift.

I wouldn't try to call this Roadmap and split in different steps yet, I think that's too early. Maybe having a "Status" paragraph suits better?